### PR TITLE
fix: fix more typos

### PR DESCRIPTION
**Pull request by UncleBill**
fixup #4226, scanning modules other than `packages/maskbook`

### CODE REVIEW

**Review by Jack-Works — DISMISSED** [review]

File: packages/dashboard/src/pages/Labs/components/PluginItem.tsx [hunk 1]
```diff
@@ -60,7 +60,7 @@ export interface PluginItemProps {
     onSetting?: (id: string) => void
 }
 
-export function PluginItemPlaceHodler() {
+export function PluginItemPlaceHolder() {
```
Comment (guanbinrui) [inline packages/dashboard/src/pages/Labs/components/PluginItem.tsx:63]:
```suggestion
export function PluginItemPlaceholder() {
```

File: packages/web3-contracts/abis/GoodGhostingPolygon.json [hunk 2]
```diff
@@ -763,7 +763,7 @@
...
...\":{\"keccak256\":\"0xcc78a17dd88fa5a2edc60c8489e2f405c0913b377216a5b26b35656b2d0dab52\",\"license\":\"MIT\",\"urls\":[\"bzz-raw://526dc85e1f9b9b45830e202568d267d93dde7a4fcccf4ad7798dadcd92304d3c\",\"dweb:/ipfs/QmaoXMB972J3cSDLtBq3xBo4jLwqD2uzXTwujtSPqkYVhR\"]},\"@openzeppelin/contracts/token/ERC20/IERC20.sol\":{\"keccak256\":\"0x5f02220344881ce43204ae4a6281145a67bc52c2bb1290a791857df3d19d78f5\",\"license\":\"MIT\",\"urls\":[\"bzz-raw://24427744bd3e6cb73c17010119af12a318289c0253a4d9acb8576c9fb3797b08\",\"dweb:/ipfs/QmTLDqpKRBuxGxRAmjgXt9AkXyACW3MtKzi7PYjm5iMfGC\"]},\"@openzeppelin/contracts/utils/Context.sol\":{\"keccak256\":\"0x8d3cb350f04ff49cfb10aef08d87f19dcbaecc8027b0bed12f3275cd12f38cf0\",\"license\":\"MIT\",\"urls\":[\"bzz-raw://ded47ec7c96750f9bd04bbbc84f659992d4ba901cb7b532a52cd468272cf378f\",\"dweb:/ipfs/QmfBrGtQP7rZEqEg6Wz6jh2N2Kukpj1z5v3CGWmAqrzm96\"]},\"@openzeppelin/contracts/utils/Pausable.sol\":{\"keccak256\":\"0x212fb1b1d4beaf74354dad9bc329f44ee3c5375ef1c32acff76b4ecefc10f1d8\",\"license\":\"MIT\",\"urls\":[\"bzz-raw://d21c68cb321d1c8d0fa39fd9ecb6bbe3b2f26623b0f38af280a010c916c85f23\",\"dweb:/ipfs/Qmf2P51HRC4ekDHLYfbXu5SXR33gXrWtq6oKSmfyWqVRuC\"]}},\"version\":1}",
```
Comment (guanbinrui) [inline packages/web3-contracts/abis/GoodGhostingPolygon.json:766]:
```suggestion
```

File: packages/web3-contracts/abis/GoodGhostingPolygon.json [hunk 3]
```diff
@@ -763,7 +763,7 @@
...
...52323020746f6b656e7320746f2061646d696e4465706f73697420617661696c61626c65206f6e6c79206265747765656e207365676d656e74203120616e64207365676d656e74206e2d31202870656e756c74696d6174652946756e6473206e6f742072656465656d65642066726f6d2065787465726e616c20706f6f6c4661696c20746f207472616e7366657220455243323020746f6b656e73206f6e207769746864726177536166654d6174683a206d756c7469706c69636174696f6e206f766572666c6f774661696c20746f207472616e736665722045524332302072657761726473206f6e207769746864726177596f75206e65656420746f206861766520616c6c6f77616e636520746f20646f207472616e7366657220444149206f6e2074686520736d61727420636f6e7472616374506c6179657220616c72656164792077697468647261772066726f6d2067616d6543616e6e6f74206a6f696e207468652067616d65206d6f7265207468616e206f6e6365506c6179657220616c726561647920706169642063757272656e74207365676d656e74506c61796572206469646e277420706179207468652070726576696f7573207365676d656e74202d2067616d65206f76657221a2646970667358221220e8c478cdc43406215b9e5d8f2b78b3c5f942400935b5089b8b9603572c862fa664736f6c634300060b00335f6d6178506c6179657273436f756e74206d7573742062652067726561746572207468616e207a65726f4661696c20746f20617070726f766520616c6c6f77616e636520746f206c656e64696e6720706f6f6c",
```
Comment (guanbinrui) [inline packages/web3-contracts/abis/GoodGhostingPolygon.json:767]:
```suggestion
```

File: packages/web3-contracts/abis/GoodGhostingPolygon.json (hunk 2 above)
Comment (guanbinrui) [reply packages/web3-contracts/abis/GoodGhostingPolygon.json:766]: I will fix it in another PR.

**Review by guanbinrui — DISMISSED** [review]

**Review by guanbinrui — APPROVED** [review]

**Review by Jack-Works — APPROVED** [review]

**Pull request merged**